examples/jobs/passjob_custom.py

```diff
@@ -12,9 +12,11 @@
 # creating tests suites for you).
 
 suite1 = TestSuite(config=config,
-                   tests=[Task("task1", Runnable("noop", "noop"))], name='suite1')
+                   tests=[Task("task1", Runnable("noop", "noop"),
+                               ['127.0.0.1:8888'])], name='suite1')
 suite2 = TestSuite(config=config,
-                   tests=[Task("task2", Runnable("noop", "noop"))], name='suite2')
+                   tests=[Task("task2", Runnable("noop", "noop"),
+                               ['127.0.0.1:8888'])], name='suite2')
```

Comment:
Not sure if this is the right place @richtja. I'm just thinking: Why not getting from default values registered and having this as default during Task initialization? Any reason?

Reply:
I think, for this example, the code is fine. It shows different uses of the status server argument. A use case I see is: the user may want to define the status server address to connect to it and listen to the messages from other processes/applications.

Reply:
For some reason, when I looked at the main page of this PR, on "Conversation" tab, it did not show the first status server definition, at line 16. Maybe my previous comment does not fit here. It would fit if the first task does not have a status server parameter set and the second task has it set.